.github/workflows/tests.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         if: startsWith(matrix.os, 'windows') == false
         run: |
           if ! git diff --exit-code --name-status; then
-            echo "::error file=$(git diff --name-status | head -n1)::PLEASE RUN bin/unipoints build"
+            echo "::error file=$(git diff --name-status | head -n 1)::PLEASE RUN bin/unipoints build"
```

Comment:
Mechele's coming from decades of posix scripting, and he's always written short options in the most compact form